operations/iuf/stages/management_nodes_rollout.md

```diff
@@ -1,7 +1,19 @@
 # management-nodes-rollout
 
-The `management-nodes-rollout` stage performs a controlled rebuild of the management worker nodes in order to reboot them to a new image and configuration. IUF will account for the necessary minimum number of critical software
-instances running on worker nodes to ensure the `management-nodes-rollout` stage operates without impacting software availability.
+The `management-nodes-rollout` stage performs a controlled rebuild of the management NCNs in order to reboot them to a new image and configuration. IUF will account for the necessary minimum number of critical software
```

Comment:
maybe define rebuild.
maybe add warning that this wipes the nodes.

Reply:
Addressed in next commit.